pitch.md

```diff
@@ -23,38 +58,137 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+
+Some errors might be the user not providing enough info for a single motion. 
+This is a problem because I want the motions to have enough information to be 
+unambiguous. In this case, I would have to display error messages that state 
+exactly what information is missing.
+
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to describe physical movement of the hands, glowsticks, and 
+body/torso. It should also be easy to string these notations together in a 
+sequence.
+
+You can attempt to simulate your physics homework, but there are better (more 
+general) programs for that.
+
+It should be impossible or very difficult to do my Algs homework, compute 
+mathematical expressions, create computer programs (in the traditional sense), 
+and make sounds.
+
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+In my 
+[free-2.md](https://github.com/tiffanyfong/project-ideas/blob/master/free-2.md), 
+I listed two DSLs on juggling and dance notation, which are technically 
+different domains from glowsticking. However, they are languages because you can 
+compose simple motions to create routines.
+
+I recently found some simulators for poi, which I consider the same domain as 
+glowsticking. 
+
+[Flower Simulator](http://www.rationalpeople.us/double.html) is a great tool for 
+visualizing a specific move (flower) in 2D.
+
+[VisualSpinner3D](http://www.spinmorepoi.com/theory/) 
+looks promising, but it doesn't show on my computer :(. Critique group: does it 
+work on your computer?
+
+Both simulators are cool, but they are more on the systems side rather than the 
+language side because they aren't very composable. You need to give a name for a 
+trick instead of my idea: connecting motions to create a trick. Moreover, the 
+tricks available are limited to what has already been premade/hard-coded. But 
+they are good at animating individual moves.
+
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+As long as I focus on the composition of glowsticking moves, I will spend a lot 
+of time on language design. My syntax isn't finalized, since there may be better 
+ways to describe circular movements. So figuring out the syntax and semantics 
+will take up a lot of time.
+
+Ideally, I would try to connect my DSL to the poi simulators, which may involve 
```

Comment:
It would be really cool if you got this to work, but I also think it's a good idea to focus on the intermediate representation first. Although I wonder how helpful it would be for the user to create a text file of motions, and get back another text file. I feel like there should be a visualization of sorts, though I'm not sure how feasible it is to do this either with the poi simulators or a different program.